templates/open-design/get-started-designers.html

```diff
@@ -0,0 +1,345 @@
+              </p>
+              <p>
+                If there are no specific tickets or bug requests, but you notice something you want to raise, do bring it to the maintainers before offering a solution!
+              </p>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="p-section--shallow">
+          <div class="row" role="listitem">
+            <div class="col-3 col-medium-3">
+              <p class="p-heading--5">8.&nbsp;&nbsp;&nbsp;Have fun and be patient</p>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Contributing to projects can be rewarding and exciting, but also chaotic and time-consuming. Most projects are maintained by passionate volunteers, and it might take time to get feedback. Validation processes can also be lengthy and your submission might not make it to the final product. Regardless, the experience you will gain from participating is a positive in and of itself.
+              </p>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="p-section--shallow">
+          <div class="row" role="listitem">
+            <div class="col-3 col-medium-3">
+              <p class="p-heading--5">9.&nbsp;&nbsp;&nbsp;Know your worth</p>
+            </div>
+            <div class="col-6 col-medium-3">
+              <p>
+                Do not feel pressured to give more than you are comfortable with. Keep in mind that your time and expertise is valuable, and should not be exploited without you getting something in return, be it material for your portfolio or the satisfaction of doing something good for your community.
+              </p>
+              <p>
+                You will likely have to educate other people on the value of design, and that's ok! Don't hesitate to ask for what you need.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="p-rule--highlight is-fixed-width" />
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <h2>What people are saying</h2>
+      </div>
+    </div>
+    <div class="row--25-75">
+      <div class="col">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-6">
+            <p role="blockquote">
+              <i>"Bringing in diverse perspectives and experiences is important for accessibility. Designers can help translate insights of people from different backgrounds to improve."</i>
+            </p>
+          </div>
+          <div class="col-3">
+            <p>Pentera maintainer</p>
```

Comment:
Copy's different on the doc but not sure which one is correct
```suggestion
            <p class="u-text--muted">From our survey</p>
```